pkg/streams/streamsDelete.go

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package streams
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"github.com/galasa-dev/cli/pkg/embedded"
+	galasaErrors "github.com/galasa-dev/cli/pkg/errors"
+	"github.com/galasa-dev/cli/pkg/galasaapi"
+	"github.com/galasa-dev/cli/pkg/spi"
+)
+
+func DeleteStream(streamName string, apiClient *galasaapi.APIClient, byteReader spi.ByteReader) error {
+
+	streams, err := getStreamsFromRestApi(streamName, apiClient, byteReader)
```

Comment:
Would be great to validate the stream name before passing it up to the API server.

Also, I don't think we need to get the stream then delete it, since the DELETE request just expects a stream name.